include/wx/charts/wxmath2dplotctrl.h

```diff
@@ -72,6 +72,9 @@ class wxMath2DPlotCtrl : public wxChartCtrl
 private:
     wxMath2DPlot m_math2dPlot;
     wxMenu m_contextMenu;
+    wxMenu *m_subMenu;
```

Comment:
You use AppendSubMenu to add the menu to m_contextMenu so m_contextMenu takes ownership. So it's correct it needs to be a pointer and m_contextMenu will delete it when it is no longer needed.